target_goal:
  # standby: {pose: [4.79713324226, 0.944046656024, 0.0], orientation: [0.0, 0.0, -0.685952058922, 0.727646736309]}
  standby: {pose: [4.85391124112, 1.20690110145, 0.0], orientation: [0.0, 0.0, -0.693556522, 0.720402214594]}
  pick: {pose: [2.61480935704, 0.31569885419, 0.0], orientation: [0.0, 0.0, 0.709, 0.705]}
  pre_pick: {pose: [2.58252527373, -0.240240165256, 0.0], orientation: [0.0, 0.0, 0.709, 0.705]}
  # pick: {pose: [2.67534509422, 0.201885666632, 0.0], orientation: [0.0, 0.0, 0.709936824382, 0.704265365744]}
  place_1: {pose: [5.05867746263, -2.83449102168, 0.0], orientation: [0.0, 0.0, -0.705392973041, 0.708816445622]} 
  place_2: {pose: [-2.48532057629, 9.4547052151, 0.0], orientation: [0.0, 0.0, 0.055728310935, 0.998445970176]} 
  place_3: {pose: [-2.48532057629, 9.4547052151, 0.0], orientation: [0.0, 0.0, 0.055728310935, 0.998445970176]} 
arm_pose:
  standby: [1.633682748037428, 3.8484475385380796, 0.664377332403468, 5.746255812428786, 4.686240999537856, 0.013789091828292604]
  pre_place: [1.708201856921426, 2.451008085005237, 5.591980488377151, 2.1399074601582133, 4.669752303964739, -0.02545106487907388]
  drink_1: [4.318998009434763, 2.154619161493975, 4.905431940424609, 1.246678170539566, 4.641360876215622, 6.001312076347856]
  drink_2: [3.4867077045115265, 2.1402833653460784, 4.723771330883689, 0.4660301544204375, 4.250704705389098, 6.113903070385354]
  follow_1: [3.51177124616248, 2.8323336046365597, 4.792249724625114, 0.3388715360621516, 3.705581729228561, -0.26743204008369426]
  follow_2: [3.5338280789427348, 2.679259366236549, 5.1520039057429035, 0.3434728459250496, 3.9626558703055825, -0.2668717442798552]
  head_ready: [3.555024444374784, 2.925448102673981, 5.497535208743017, 0.4005828041841721, 4.015191782813466, -0.27277958432878935]

box_information:
  box_1: {arm_pose: [-1.1147592183117907, 4.029135295332794, 0.7994051257297473, 2.0006133493446545, 4.658114822633311, -0.15615986006276036], height: 0.0, size: 0.6, id: 101}
  box_2: {arm_pose: [1.0790958294691309, 4.033414995204552, 0.6454974510665965, 4.3939580046372235, 4.628563321913356, 0.21585062115525136], height: 0.0, size: 0.6, id: 102}
  box_3: {arm_pose: [1.404615074773952, 4.048947349630221, 0.8375142951385086, 4.454578475266261, 4.640331830770416, 0.06605971916179391], height: 0.0, size: 0.6, id: 103}
  box_4: {arm_pose: [-1.1642402853122722, 3.359815002788085, 0.7830801479846433, 1.832967491882221, 4.475351131768879, 0.46011241074119963], height: 0.1, size: 0.6, id: 104}
  box_5: {arm_pose: [1.184603081323127, 3.229917722722816, 0.6642034945633131, 4.527024446399687, 4.483185619063675, -0.6137732769503274], height: 0.1, size: 0.6, id: 105}
  box_6: {arm_pose: [1.3930335198254156, 3.4857532275768426, 0.9504544961739433, 4.548079930610071, 4.519393424612174, -0.64322817150174], height: 0.1, size: 0.6, id: 106}
  box_7: {arm_pose: [-1.2599521643608478, 3.3132706701006613, 0.9473047089736794, 1.7199401389001137, 4.553844076473438, 0.698942367033552], height: 0.41, size: 0.6, id: 107}
  box_8: {arm_pose: [1.1429299365855672, 3.4050061833256393, 0.9799717086948637, 4.6367323022410645, 4.597201936916583, -0.7931578824446394], height: 0.41, size: 0.6, id: 108}
  box_9: {arm_pose: [1.381543977092541, 3.405926571798371, 1.0443784620246943, 4.54811242117537, 4.5180325492951425, -0.7930339788749236], height: 0.41, size: 0.6, id: 109}

guide_goal:
  [
    {name: "1", pose: [4.767, 1.246, 0.0], orientation: [0.0, 0.0, -0.693, 0.721], file_path: "/home/caster/caster_moma_ws/src/caster_moma_ros/caster_moma_app/audio/home.wav"},
    # {name: "1", pose: [2.6196938538, 0.489361252368, 0.0], orientation: [0.0, 0.0, -0.413566638887, 0.910473852013], file_path: "/home/caster/caster_moma_ws/src/caster_moma_app/audio/home.wav"},
    {name: "2", pose: [3.8350672024, 0.440261080078, 0.0], orientation: [0.0, 0.0, -0.77636939156, 0.630278167042], file_path: "/home/caster/caster_moma_ws/src/caster_moma_ros/caster_moma_app/audio/drugs.wav"},
    {name: "3", pose: [5.09789428027, 0.176737970335, 0.0], orientation: [0.0, 0.0, -0.884615210637, 0.466321701308], file_path: "/home/caster/caster_moma_ws/src/caster_moma_ros/caster_moma_app/audio/black.wav"} ,
    {name: "4", pose: [4.46113934853, -2.66959002363, 0.0], orientation: [0.0, 0.0, -0.862046865418, 0.506828572421], file_path: "/home/caster/caster_moma_ws/src/caster_moma_ros/caster_moma_app/audio/theatre.wav"} ,
    {name: "5", pose: [2.97861198733, -5.8541323724, 0.0], orientation: [0.0, 0.0, -0.7102623227, 0.703937094457], file_path: "/home/caster/caster_moma_ws/src/caster_moma_ros/caster_moma_app/audio/vrar.wav"},
  ]
